src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/entity/CommunityOffenderManager.kt

```diff
@@ -30,14 +29,5 @@ data class CommunityOffenderManager(
 
   val lastName: String?,
 
-  @OneToMany(mappedBy = "responsibleCom")
-  val licencesResponsibleFor: List<Licence> = emptyList(),
-
-  @OneToMany(mappedBy = "createdBy")
-  val licencesCreated: List<Licence> = emptyList(),
-
-  @OneToMany(mappedBy = "submittedBy")
-  val licencesSubmitted: List<Licence> = emptyList(),
-
```

Comment:
These backward relations aren't currently used so removed (can always add in if necessary)